plugins/Jobs.cpp

```diff
@@ -1,6 +1,62 @@
 #include "Jobs.h"
 #include "../BedrockServer.h"
 
+#undef SLOGPREFIX
+#define SLOGPREFIX "{DupeDiagnoser} "
+class DupeDiagnoser {
+  public:
+    DupeDiagnoser();
+    void check(uint64_t jobID, string requestID);
+
+  private:
+    mutex _m;
+    map<uint64_t, pair<string, chrono::steady_clock::time_point>> _jobIdToRequestIdAndInsertionTimeMap;
+    multimap<chrono::steady_clock::time_point, uint64_t> _insertionTimeToJobIdMap;
+};
+
+DupeDiagnoser::DupeDiagnoser() {
+}
```

Comment:
why is this class definition not in the header?

Reply:
Cause it's throwaway code and I want to delete it as soon as we fix this.